test/ZxcvbnTest.php

```diff
@@ -71,14 +71,14 @@ public function sanityCheckDataProvider()
         return [
             ['password', 0, ['dictionary',], 'less than a second', 3],
             ['65432', 0, ['sequence',], 'less than a second', 101],
-            ['sdfgsdfg', 1, ['repeat',], 'less than a second', 2595.0000000276],
+            ['sdfgsdfg', 1, ['repeat',], 'less than a second', 2595],
             ['fortitude', 1, ['dictionary',], '1 second', 11308],
             ['dfjkym', 1, ['bruteforce',], '2 minutes', 1000001],
             ['fortitude22', 2, ['dictionary', 'repeat',], '2 minutes', 1140700],
             ['absoluteadnap', 2, ['dictionary', 'dictionary',], '25 minutes', 15187504],
             ['knifeandspoon', 3, ['dictionary', 'dictionary', 'dictionary'], '1 day', 1108057600],
-            ['h1dden_26191', 3, ['dictionary', 'bruteforce', 'date'], '3 days', 2642940400],
-            ['4rfv1236yhn!', 4, ['spatial', 'sequence', 'bruteforce'], '1 month', 38980000000.414],
+            ['h1dden_26191', 3, ['dictionary', 'bruteforce', 'date'], '3 days', 2730628000], // Failed asserting that 2730628000 matches expected 2642940400.
```

Comment:
I can not for the life of me figure out how this test is in a failed state. Github Actions passed [on 2021-11-30](https://github.com/bjeavons/zxcvbn-php/actions/runs/392046040), and travis passed [on 2021-11-25](https://travis-ci.org/github/bjeavons/zxcvbn-php/jobs/745961950) but an RPM packager at least as far back as [June 2020](https://github.com/hrnciar/rpm-specs/blame/7224a58f5ff3a0f6879313766965c49c91250010/rpm-specs/php-mkopinsky-zxcvbn-php.spec#L111) had to disable that test. If I check out old commits now, they all fail.

I guess at this point we should just use the new value as you have, and we can get rid of that comment.

Reply:
I tried resetting the code back a few versions and re-tried the tests. I suspect there might be something leaking in that is platform specific.